src/tools/base/BaseTool.js

```diff
@@ -43,14 +44,19 @@ class BaseTool {
       defaultStrategy || Object.keys(this.strategies)[0] || undefined;
     this.activeStrategy = this.defaultStrategy;
 
+    if (svgCursor) {
+      this.svgCursor = svgCursor;
+    }
+
     // Options are set when a tool is added, during a "mode" change,
     // Or via a tool's option's setter
     this._options = {};
     // Configuration is set at tool initalization
     this._configuration = Object.assign({}, configuration);
 
-    // True if tool has a custom cursor, causes the frame to render on every mouse move when the tool is active.
-    this.hasCursor = false;
+    // True if tool has a canvas cursor, causes the frame to render on every mouse move when the tool is active.
+    this.hasCanvasCursor = false;
+    this.hideDefaultCursor = false;
```

Comment:
Would we want these values to be set via configuration on init? Not setting them based on incoming config means we can't set them as part of the `globalToolSync`.

Reply:
Hmm, any tool so far that has a canvas cursor requires it operationally. At some level its just a flag to signal that the screen should re-render on mouse move. `hasCanvasCursor` existed prior to `svgCursor`s, so perhaps its should just be named 'updateOnMouseMove`?

I think the tool should hard set this based on its requirements. I'm happy to discuss, but being able to switch the brush cursor or the sculpter cursor off just seems like a bad idea to me.